server/models/Profile.js

```diff
@@ -0,0 +1,45 @@
+const mongoose = require('mongoose');
+
+const profileSchema = new mongoose.Schema({
+    firstName: {
+        type: String,
+        required: true,
+        trim: true
+    },
+    lastName: {
+        type: String,
+        required: true,
+        trim: true
+    },
+    description: String,
+    gender: {
+        type: String,
+        required: true,
+        enum: ['Male', 'Female']
+    },
+    birthDate: Date,
+    email: {
+        type: String,
+        lowercase: true,
+        required: true,
+        trim: true,
+        unique: true
+    },
+    phoneNumber: {
+        type: Number,
+        trim: true
+    },
+    address: {
+        type: String,
+        required: true
+    },
+    availability: [
+        {
+            from: Date,
+            to: Date
+        }
+    ],
```

Comment:
This looks decent but I'd like to clarify things... Would you mind sharing your thoughts that went into implementing this? (e.g. Why did you choose this structure? How is the API going to work? etc.)